kuma/wiki/forms.py

```diff
@@ -327,63 +356,57 @@ def clean_current_rev(self):
             # If there's no document yet, just bail.
             return current_rev
 
-    def save_section(self, creator, document, **kwargs):
```

Comment:
Yeah, I think we should remove it since it makes the editing flow still quite cumbersome. Have we agreed that saving sections only doesn't make sense? I've never seen the feature in the frontend, does it even exist there?